codesets/mlflow/keras/train.py

```diff
@@ -5,14 +5,34 @@
 os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
 import tensorflow as tf
 
-from tensorflow.keras import datasets, layers, models
+from tensorflow.keras import datasets, layers, models, optimizers
 
 
+def get_mkl_enabled_flag():
+
+    mkl_enabled = False
+    major_version = int(tf.__version__.split(".")[0])
+    minor_version = int(tf.__version__.split(".")[1])
+    if major_version >= 2:
+        if minor_version < 5:
+            from tensorflow.python import _pywrap_util_port
+        else:
+            from tensorflow.python.util import _pywrap_util_port
+            onednn_enabled = int(os.environ.get('TF_ENABLE_ONEDNN_OPTS', '0'))
+        mkl_enabled = _pywrap_util_port.IsMklEnabled() or (onednn_enabled == 1)
+    else:
+        mkl_enabled = tf.pywrap_tensorflow.IsMklEnabled()
+    return mkl_enabled
+
+print ("We are using Tensorflow version", tf.__version__)
+print("MKL enabled :", get_mkl_enabled_flag())
```

Comment:
I assume this code is coming from the intel example pages of how to test the MKL enablement: could you add at least a comment to the function recognizing the source?

Also, this is really tiny bit, but could  you align the whitespace for those print methods on lines 27-28 ?

Reply:
done